### ISSUE TITLE
Pb with the CityJSON transform

### ISSUE THREAD

**Issue by alavenant** [citygml4j/citygml4j#39]
This code read a citygml file and write it in cityjson 

```java
	public static void test()  {

    	File f = new File("myFile.GML");

        CityGMLContext cityGMLContext = CityGMLContext.newInstance();

        CityGMLInputFactory in = cityGMLContext.createCityGMLInputFactory();

        AbstractFeature feature;
        try (CityGMLReader reader = in.createCityGMLReader(f.toPath())) {
            feature = reader.next();
        }

        CityJSONContext cityJSONContext = CityJSONContext.newInstance();

        CityJSONVersion version = CityJSONVersion.v1_1;
        CityJSONOutputFactory out = cityJSONContext.createCityJSONOutputFactory(version)
                .withVertexPrecision(3)
                .withTextureVertexPrecision(5)
                .applyTransformation(true);

    	File fo = new File("myFile.Out.JSON");

        try (CityJSONWriter writer = out.createCityJSONWriter(fo.toPath())) {
            writer.withMetadata(metadata)
                    .writeCityObject(feature);
        }

	}
```

The cityjsonn result has the transform : 

```json
"transform":{"scale":[0.001,0.001,0.001],"translate":[521211.074,6456397.886,90.711]}
```

But the coordinates (521242.73015 6456434.20157 95.09962) comes to (9246,4308959777,6289). It's seems to be good for X and Z. But not for the Y.

[Zone_1.GML.zip](https://github.com/citygml4j/citygml4j/files/11734110/Zone_1.GML.zip)

**Comment by clausnagel** [comment]
Are you sure you are working with the latest 3.0.0 release? https://github.com/citygml4j/citygml4j/releases/tag/v3.0.0

There was a bug with the transform in pre-releases and a fix in 3.0.0 (see [changelog](https://github.com/citygml4j/citygml4j/blob/master/CHANGELOG.md)). If you work with a pre-release, then first update to 3.0.0.

**Comment by alavenant** [comment]
ok ! It's works with the 3.0.0. I was on the 3.0.0 -rc5. 
Thanks
